application/client/src/app/ui/views/toolbar/search/input/input.ts

```diff
@@ -23,12 +23,14 @@ export class SearchInput {
         accept: Subject<string>;
         recent: Subject<void>;
         edit: Subject<void>;
+        reaccept: Subject<void>;
     } = {
         drop: new Subject(),
         clear: new Subject(),
         accept: new Subject(),
         recent: new Subject(),
         edit: new Subject(),
+        reaccept: new Subject(),
```

Comment:
motivation of event `reaccept` isn't clean

Reply:
Was for updating search results once the flags are changed. The code has been adjusted and behavior for it is included into `accept` instead